Marlin/src/inc/SanityCheck.h

```diff
@@ -1544,6 +1544,15 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
   #endif
 #endif
 
+/**
+ * Make sure NO_MOTION_BEFORE_HOMING if NO_MOTION_BEFORE_HOMING enabled
```

Comment:
`Make sure NO_MOTION_BEFORE_HOMING if NO_MOTION_BEFORE_HOMING enabled` should be: `HOME_AFTER_DEACTIVATE needs NO_MOTION_BEFORE_HOMING enabled`